src/main/java/com/uber/cadence/internal/worker/TaskListKind.java

```diff
@@ -0,0 +1,33 @@
+/*
+ *  Copyright 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ *  use this file except in compliance with the License. A copy of the License is
+ *  located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package com.uber.cadence.internal.worker;
+
+public enum TaskListKind {
+  TASK_LIST_KIND_NORMAL(0),
+  TASK_LIST_KIND_STICKY(1);
+
+  private final int value;
+
+  TaskListKind(int value) {
+    this.value = value;
+  }
+
+  public com.uber.cadence.TaskListKind toThrift() {
+    return com.uber.cadence.TaskListKind.findByValue(this.value);
```

Comment:
I'm sure this is very borderline value for now, but in case some enums get out of sync or something, unit-test coverage?

Reply:
right. added unit test